LaTeX/syntax_test_xparse.tex

```diff
@@ -0,0 +1,230 @@
+% SYNTAX TEST "Packages/LaTeX/LaTeX.sublime-syntax"
+
+% <- text.tex.latex
+
+\NewDocumentCommand{\noparams}{}{content}
+%^^^^^^^^^^^^^^^^^^ meta.function.latex keyword.declaration.function.latex storage.modifier.newcommand.latex
+%                  ^^^^^^^^^^^ meta.function.identifier.latex
+%                  ^ punctuation.definition.group.brace.begin.latex
+%                   ^^^^^^^^^ entity.name.newcommand.latex
+%                   ^ punctuation.definition.backslash.latex
+%                            ^ punctuation.definition.group.brace.end.latex
+%                             ^^ meta.function.parameters.latex
+%                             ^ punctuation.definition.group.brace.begin.latex
+%                              ^ punctuation.definition.group.brace.end.latex
+%                               ^^^^^^^^^ meta.function.body.tex meta.group.brace.tex
+%                               ^ punctuation.definition.group.brace.begin.tex
+%                                       ^ punctuation.definition.group.brace.end.tex
```

Comment:
Not a big deal, but it is sometimes useful to group meta scope tests and "normal" scope tests especially if checking boundaries etc. is of interest.

```suggestion
%^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ - meta.function meta.function
%^^^^^^^^^^^^^^^^^^ meta.function.latex keyword.declaration.function.latex storage.modifier.newcommand.latex
%                  ^^^^^^^^^^^ meta.function.identifier.latex
%                             ^^ meta.function.parameters.latex
%                               ^^^^^^^^^ meta.function.body.tex meta.group.brace.tex
%                                        ^ - meta.function
%                  ^ punctuation.definition.group.brace.begin.latex
%                   ^^^^^^^^^ entity.name.newcommand.latex
%                   ^ punctuation.definition.backslash.latex
%                            ^ punctuation.definition.group.brace.end.latex
%                             ^ punctuation.definition.group.brace.begin.latex
%                              ^ punctuation.definition.group.brace.end.latex
%                               ^ punctuation.definition.group.brace.begin.tex
%                                       ^ punctuation.definition.group.brace.end.tex
```